### ISSUE TITLE
Class assignments/tests page

### ISSUE THREAD

**Issue by kmaus** [mikevo/uno-heterogeneous-knowledge#17]
When a class is selected, a list of assignments/test should be available. Instructors should be able to add new assignments, make existing ones visible/hidden[nicety], and edit assignments. New and edit may lead to the same/similar views. 

Students should get a list of completed and due assignments. Completed should have a grade associated with it(or grade pending) and maybe a review option(could also be toggled by the instructor).

**Comment by mikevo** [comment]
A due to date/time would be nice.

**Comment by kmaus** [comment]
Yeah, it should be listed definitely be listed on this view. Do we want to be able to change the date from this view or the edit view?
